sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -940,7 +948,7 @@ object SparkSession {
         }
 
         session = new SparkSession(sparkContext, None, None, extensions)
-        options.foreach { case (k, v) => session.sessionState.conf.setConfString(k, v) }
+        options.foreach { case (k, v) => session.initialSessionOptions.put(k, v) }
```

Comment:
Thank you for review, @cloud-fan . You mean delete this simply, right?
That would be great.